### DISCUSSION TITLE
I can't login to Copilot in PyCharm

### DISCUSSION THREAD

**Discussion by nahco314** [community/community#30673] [Copilot Conversations] [answered]
I am using PyCharm Professional 2022.2.1 on Kubuntu 22.04 and the version of the Copilot plugin is 1.1.30.1887.

When I try to log in to Copilot from PyCharm and put in the code with Copy and Open, the browser shows "Congratulations, you're all set!", but after waiting a while, I get PyCharm's "Wating for GitHub Authentication..." disappears and ends without any dialog indicating success.
After that, when I check, the login is not working.

I have tried restarting the IDE/PC, reinstalling the IDE/plugins, and changing versions, but no change.

By the way, I was able to log in correctly before.

**Comment by davecheney** [comment]
@nahco314 as a technical preview user you received complementary access to copilot until August the 22nd. If you wish to continue to use Copilot, you can sign up for a subscription plan here , https://github.com/github-copilot/signup

**Comment by nahco314** [comment, marked as answer]
I see, I had assumed it was ok because I had registered with github education, but it seems I needed to do a little configuration.
I set it up properly and tried again and it worked, thanks!

**Reply by davecheney** [reply]
Correct, if you have a student coupon, apply that to your account then the complementary copilot option will be available during signup.
